cou/steps/analyze.py

```diff
@@ -55,6 +59,42 @@ def __post_init__(self) -> None:
         self.current_cloud_o7k_release = self._get_minimum_cloud_o7k_release()
         self.current_cloud_series = self._get_minimum_cloud_series()
 
+    @property
+    def apps_control_plane(self) -> list[OpenStackApplication]:
+        """Return list of control plane applications.
+
+        :return: Control plane application lists.
+        :rtype: list[OpenStackApplication]
+        """
+        control_plane, _ = self._split_apps(self.apps)
+        return control_plane
+
+    @property
+    def apps_data_plane(self) -> list[OpenStackApplication]:
+        """Return list of data plane applications.
```

Comment:
Could we add a description to explain why OVN is not included in the expected output? Users might misuse this function if they only read the description. 

Another option is to exclude OVN by the caller. (out side of this function)

Reply:
It will be nice we filter ovn-chassis here. It will be more easy to maintain in the future.

Reply:
Updated